transportable-udfs-examples/transportable-udfs-example-udfs/build.gradle

```diff
@@ -12,6 +12,7 @@ dependencies {
   implementation('com.google.guava:guava:24.1-jre')
   implementation('org.apache.commons:commons-io:1.3.2')
   testImplementation('com.linkedin.transport:transportable-udfs-test-spi')
+  testImplementation('io.airlift:aircompressor:0.21')
```

Comment:
Why do we need it?

Reply:
This change is to make `transportabl-udfs-examples` work. As there are some unit tests running against UDFs generated for Trino in `transportable-udfs-example-udfs`, these tests are failing with Trion UDFs compiled with Java 17 for the following error:
```
java.lang.NoClassDefFoundError: Could not initialize class io.airlift.compress.zstd.UnsafeUtil
		at io.airlift.compress.zstd.ZstdFrameCompressor.writeMagic(ZstdFrameCompressor.java:57)
		at io.airlift.compress.zstd.ZstdFrameCompressor.compress(ZstdFrameCompressor.java:143)
io.airlift.compress.IncompatibleJvmException: Zstandard requires access to java.nio.Buffer raw address field
```
According to the investigation, this exception is thrown from the old version of the package of `aircompressor` because of the module system introduced in Java 9 and the java package “java.nio” is not open for modification. 

The solution is designate the tests to use the latest version of the package of `aircompressor` which fix this problem.